docs/gui/menus.md

```diff
@@ -320,7 +323,7 @@ public class MyMob extends Mob implements MenuProvider {
   @Override
   public InteractionResult mobInteract(Player player, InteractionHand hand) {
     if (!this.level.isClientSide && player instanceof ServerPlayer serverPlayer) {
-      NetworkHooks.openScreen(serverPlayer, this);
+      serverPlayer.openMenu(this,(extraDataWriter)->{});
```

Comment:
Ok, why do I need to send arbitrary data here? What's the point in using the byte buffer like this if I'm not sending anything? Since it's agnostic leave a comment inside and say 'send information needed to display on the client' or something similar.